components/ClaimSection.tsx

```diff
@@ -4,13 +4,20 @@ import { ScenarioContent } from '../types/common'
 
 export interface ClaimSectionProps extends ScenarioContent {
   loading: boolean
+  userArrivedFromUioMobile: boolean
 }
 
-export const ClaimSection: React.FC<ClaimSectionProps> = ({ loading = false, statusContent, detailsContent }) => {
+export const ClaimSection: React.FC<ClaimSectionProps> = ({
+  loading = false,
+  userArrivedFromUioMobile = false,
```

Comment:
I FINALLY FOUND IT!

I knew there must be a way to avoid this pass-pass-pass through the components:

> Context is designed to share data that can be considered “global” for a tree of React components, such as the current authenticated user, theme, or preferred language. For example, in the code below we manually thread through a “theme” prop in order to style the Button component

[React Context](https://reactjs.org/docs/context.html)

Reply:
to be clear: if we do this I suggest we do this as a follow on, not in this PR